test/export/test_export.py

```diff
@@ -1299,6 +1299,17 @@ def false_fn(x):
         M()(torch.randn(7))
         torch.export.export(M(), (torch.randn(7),), strict=strict)
 
+    def test_cond_branches_return_constant_int(self):
+        class M(torch.nn.Module):
+            def forward(self, x):
+                idx = torch.cond(x.sum() > 3, lambda: 0, lambda: 1, tuple())
+                return x[idx]
+
+        args = (torch.randn(3, 3),)
+        m = M()
+        ep = export(M(), args)
+        self.assertEqual(m(*args), ep.module()(*args))
+
```

Comment:
Can we get a self.assertEquals on the graph? I'm not convinced that this doesn't just bake a branch in the graph lol

Reply:
sure lol, added the graph check.